torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2194,6 +2194,19 @@ def _make_tensor_helper(shape, low=None, high=None):
 
     return samples
 
+def sample_inputs_kron(op_info, device, dtype, requires_grad):
+    test_cases = (
+        (S, S),
```

Comment:
kron is a binary operation (https://pytorch.org/docs/master/generated/torch.kron.html?highlight=kron#torch.kron), so the test case is both tensors. Following the current method_tests() entry, it would be:

```
test_cases = (
((S, S), (M, L)),
)
```

Reply:
Done. Should we still use (S, M) for the second operator? On my machine, this reduces the running time for the affected tests from 136 seconds to 40.